packages/ubuntu_wsl_setup/lib/pages/applying_changes/applying_changes_model.dart

```diff
@@ -0,0 +1,19 @@
+import 'package:subiquity_client/subiquity_client.dart';
+
+import '../../installing_state.dart';
+
+/// Implements the business logic of the WSL Applying Changes Page.
+///
+/// See also:
+///  * [ApplyingChangesPage]
+class ApplyingChangesModel {
+  /// Creates a model for the 'applying changes' page.
+  ApplyingChangesModel(this._monitor) {
+    _isInstalling = _monitor.onStatusChanged
```

Comment:
moving this from the body to the initializer list should allow removing "late"

Reply:
Great tip was well. Even got rid of the monitor reference, since it was only used to initialize this Stream.